test-apps/ui-test-app/src/frontend/appui/imodelopen/ProjectDropdown.tsx

```diff
@@ -136,16 +126,13 @@ export class ProjectDropdown extends React.Component<ProjectDropdownProps, Proje
       <div className="pp">
         <div className="pp-content" onClick={this._splitterClicked} ref={(element) => { this._target = element; }}>
           <div>
-            <span className="number">{this.props.currentProject ? this.props.currentProject.projectNumber : ""}</span>
+            <span className="number">{this.props.currentProject ? this.props.currentProject.code : ""}</span>
             <span className="name">{this.props.currentProject ? this.props.currentProject.name : ""}</span>
           </div>
           <span className={splitterClassName} />
         </div>
         <div className="pp-highlight" />
         {this.renderDropdown()}
-        {this.state.showProjectsDialog &&

```

Comment:
Why is it necessary to remove the project dialog?

Reply:
We trimmed down the registry down to its core functionalities, as a result the Project Dialog lost relevancy as most of its features (favorites, recently used) are no longer available in the Registry, as well as the fact the Registry is no longer accessible though the UI-Framework meaning it could not use these functionalities either.

Reply:
I pulled your branch and I am unable to open an file from the hub in ui-test-app, I see no projects in my project drop down and the spinner never stops spinning.

Reply:
@calebmshafer were you able to add in the fixed project dropdown?

Reply:
Working on a fix for this now.

Reply:
@bsteinbk mind taking another look at this? I have a couple more changes I'm about to push that should smooth this out

Reply:
@calebmshafer - I pulled at built and if I choose "IModel Open" I get the following...

Unhandled Rejection (Error): {
                              "statusCode": 401,
                              "message": "Access denied due to invalid access_token. Make sure to provide a valid token for this API endpoint."
                          }

Reply:
@bsteinbk hm okay, I thought I fixed that issue. Was that with Web or Electron or both?

Reply:
It happens in both Web and Electron.

Reply:
@bsteinbk were you prompted to sign-in or did it attempt to go straight to the Project page?

Reply:
I was never prompted.  I just tried clearing cookies and local storage and I still did not get prompted to sign in.

Reply:
Just pushed a fix. It was due to my PR that changed the default issuer yesterday so it's probably broken on master at the moment too.